Doc/cmus.txt

```diff
@@ -331,79 +331,49 @@ u                       win-update
 
 @h1 LIBRARY VIEW SORTING
 
-The library view (the tree-like one; not the sorted library view, for which
-the sorting is controlled by the user by setting lib_sort - see `CONFIGURATION
-OPTIONS`), is sorted automatically by cmus using the information found in the
-tagging information provided by the audio files.
+The library view (the tree-like one; not the sorted library view, which is
+configured with lib_sort - see `CONFIGURATION OPTIONS`), is sorted automatically
+using tags from the audio files.
 
-Generally, in the library view cmus uses three levels of sorting: the first
-level would be the artist name, the second one the album and finally the
-actual track.
+Note: Albums which feature various artists (e.g. samplers or compilations) are
+treated specially. If an album artist tag or the ID3v2 *TPE2* frame is set, it
+will be used instead of the real artist name. Otherwise, cmus determines if the
+album is a compilation (if *albumartist* or *artist* are set to *Various
+Artists*, *Various*, *VA*, or *V/A*; or if *compilation* or *partofacompilation*
+are set to a truthy value; or if the ID3v2 *TCMP* frame is set). If so, the
+album is named *<Various Artists>*.
```

Comment:
I think you meant `artist`, not `album`.

Reply:
The rest LGTM.

Reply:
Fixed.